cluster-up/cluster/kind-1.19-vgpu/config_vgpu_cluster.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+[ $(id -u) -ne 0 ] && echo "FATAL: this script requires sudo privileges" >&2 && exit 1
+
+set -xe
+
+SCRIPT_PATH=$(dirname "$(realpath "$0")")
+
+source ${SCRIPT_PATH}/vgpu-node/node.sh
+echo "_kubectl: " ${_kubectl}
+echo "KUBECTL_PATH: " $KUBECTL_PATH
+echo "KUBEVIRTCI_PATH: " ${KUBEVIRTCI_PATH}
+source ${KUBEVIRTCI_PATH}/cluster/kind/common.sh
+echo "_kubectl: " ${_kubectl}
+
+worker_nodes=($(_kubectl get nodes -o custom-columns=:.metadata.name --no-headers))
```

Comment:
it get all nodes isn't it ?
not just workers as the name says,
i believe remount_sysfs should run on all, so just the name should be changed please